wazo_dird_client/__init__.py

```diff
@@ -1,7 +1,5 @@
 # -*- coding: utf-8 -*-
-# Copyright 2014-2018 The Wazo Authors  (see the AUTHORS file)
+# Copyright 2014-2022 The Wazo Authors  (see the AUTHORS file)
 # SPDX-License-Identifier: GPL-3.0-or-later
 
-from wazo_dird_client.client import DirdClient as Client
-
-__all__ = ['Client']
+from wazo_dird_client.client import DirdClient as Client  # noqa
```

Comment:
Why this change?

Reply:
It was giving me `cannot import exceptions from wazo_dird_client`; and when I change this code, the import was working.

Reply:
Done